reference/forms/types/datetime.rst

```diff
@@ -74,8 +74,10 @@ Defines the ``format`` option that will be passed down to the date field.
 See the :ref:`DateType's format option <reference-forms-type-date-format>`
 for more details.
 
-The ``date_format`` option does not have any effect when the form is rendered
-as an HTML5 datetime input.
+.. versionadded:: 4.3
```

Comment:
You should use newly introduced deprecated directive here , same below

Reply:
fixed